docs/configuring-playbook-bridge-mautrix-wsproxy.md

```diff
@@ -0,0 +1,30 @@
+# Setting up Mautrix wsproxy (optional)
+
+The playbook can install and configure [mautrix-wsproxy](https://github.com/mautrix/wsproxy) for you.
+
+See the project's [documentation](https://github.com/mautrix/wsproxy#readme) to learn what it does and why it might be useful to you.
+
+Use the following playbook configuration:
+
+```yaml
+matrix_mautrix_wsproxy_enabled: true
+matrix_mautrix_wsproxy_appservice_token: 'random string'
+matrix_mautrix_wsproxy_homeserver_token: 'random string'
```

Comment:
The playbook already sets these automatically in `group_vars/matrix_servers`. It's not necessary to set these manually.

Reply:
I first took these away, and then I got wondering: are they set when compiling imessage-mautrix? In that case I should not generate them but set them to whatever was used there. If they are generated by ansible, how do you even see what they are @spantaleev ?

The [mautrix-imessage config](https://mau.dev/mautrix/imessage/-/blob/master/example-config.yaml#L57) says this
```
    # Authentication tokens for AS <-> HS communication. Autogenerated; do not modify.
    as_token: "This value is generated when generating the registration"
    hs_token: "This value is generated when generating the registration"
```